app/managers.py

```diff
@@ -0,0 +1,32 @@
+import sqlite3
+
+from models import Actor
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self._connection = sqlite3.connect("../cinema.sqlite")
+        self.table_name = "actors"
+
+    def create(self, first_name: str, last_name: str) -> None:
+        self._connection.execute(
+            f"INSERT INTO {self.table_name} (first_name, last_name) VALUES (?, ?)",
+            (first_name,last_name))
+        self._connection.commit()
+
+    def all(self) -> list:
```

Comment:
Indicate that you return a list of `Actor` class instances in the return type annotation